addOns/openapi/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- When a definition doesn't define a response then an appropriate warning bubbles up (no longer resulting in an NullPointerException) [Issue 7115].
```

Comment:
Use parentheses instead? That's syntax for link.

Reply:
```suggestion
- When a definition doesn't define a response then an appropriate warning bubbles up, no longer resulting in an NullPointerException (Issue 7115).
```

Seem okay?

Reply:
It does.